drivers/adc/ad463x/ad463x.c

```diff
@@ -53,6 +53,19 @@
 /******************************************************************************/
 #define AD463x_TEST_DATA 0xAA
 
+#define NANO 1000000000ULL
```

Comment:
There is a NANO defined here:
https://github.com/analogdevicesinc/no-OS/blob/04d452689395136c2cd32c9a49ab1369dba9c936/include/no_os_units.h#L56